spec/lib/services/lookup-spec.js

```diff
@@ -369,7 +379,34 @@ describe('Lookup Service', function () {
 
             this.sandbox.stub(lookupService, 'ipAddressToMacAddress').resolves('00:11:22:33:44:55');
 
-            var req = { connection: { remoteAddress: '10.1.1.1' } },
+            var req = { 
+                    connection: { remoteAddress: '10.1.1.1' },
+                    get: function() {
+                        return undefined;
+                    }
+                },
+                next = function () {
+                    expect(req.macaddress).to.equal('00:11:22:33:44:55');
```

Comment:
When using `done()` I usually wrap these in a try/catch, otherwise on failure cases the test just times out rather than showing the real assertion error, e.g.:

```
next = function () {
    try {
        <expect>
        done();
    catch (e) {
        done(e);
    }
};
```